packages/types/src/index.ts

```diff
@@ -223,8 +223,8 @@ export type Step = {
   area: boolean;
   bogusName: boolean;
   distance: number;
-  elevation?: ElevationData;
-  elevationProfile?: ElevationProfileComponent[];
+  elevation: ElevationData;
+  elevationProfile: any;
```

Comment:
Ah, this was done because I before I pushed i did a git pull origin master to keep things up to date. I found that I was getting build errors for things I didn't touch. So the second commit in this pull request is all stuff that I did _after_ I did everything I needed to do to get the feature working.

* bf90e3efdbef3857f1a59df4f7571fcaf4b5203a
* 5b1f90d703e4ae4b93e8c95eced81f5d644c65fd

When I finished my work, I ran:
`
git pull origin master
`
to make sure that I'm incorporating the latest and greatest into my branch so I'm not behind. Before this, running 
`
yarn prepublish
`
yielded no problems for me.

After getting my branch up to date and trying the same, however, I ran into this error:
`
lerna ERR! yarn run tsc exited 2 in '@opentripplanner/core-utils'
lerna ERR! yarn run tsc stdout:
$ tsc
src/itinerary.ts(6,3): error TS2305: Module '"@opentripplanner/types"' has no exported member 'ElevationProfileComponent'.
src/itinerary.ts(348,12): error TS2339: Property 'elevationProfile' does not exist on type 'Step'.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
`
I hadn't originally touched any of these files.